# [line 2] Error at 'foo': Expect '(' after 'fn'.
while fn foo(){} {}
